popde/density_estimate.py

```diff
@@ -18,9 +18,39 @@ def __init__(self, data, backend='scipy', bandwidth=1., dim_names=None):
                points of the data define each kernel position
                each row is a point, each column is a parameter.
         kwargs:
-            bandwidth : The bandwidth of the kernel used for smoothing
+           bandwidth : The bandwidth of the kernel used for smoothing
             dim_names : sequence of dimension names, e.g. ('m1', 'z', 'chi_eff')
                         values must be strings
+        Example:
+        >>> #For two dimensional case
```

Comment:
Can you remove the `>>>` please, someone who wants to run the example cannot use them as input so they would have to remove them anyway